src/extension/extension.ts

```diff
@@ -24,7 +24,7 @@ export function activate(context: vscode.ExtensionContext): HttpBookApi | false
     httpyacExtension.exports.environementChanged.event(environementChanged);
 
     const httpNotebookOutputFactory = new notebook.HttpNotebookOutputFactory(config, httpyacExtension.exports.httpyac);
-
+    httpNotebookOutputFactory.initiailze();
```

Comment:
Was the missing await intentional? 
```suggestion
    await httpNotebookOutputFactory.initialize();
```

Reply:
Yes, I don't see any reason to wait do that. That'll complete very quickly, besides it'll take a bit longer for notebooks to open etc, hence no point in waiting.

Then again no harm on waiting as well. I think either is fine